content/doc/book/pipeline/pipeline-as-code.adoc

```diff
@@ -80,34 +80,23 @@ dropdowns in other Jenkins jobs.
 
 == Folder Computation
 
-_Multibranch Pipeline_ projects and _Organization Folders_ extend the existing
-folder functionality by introducing 'computed' folders.  Computed folders
-automatically run a process to manage the folder contents.  This computation, in
-the case of _Multibranch Pipeline_ projects, creates child items for each
-eligible branch within the child.  For _Organization Folders_, computation
-populates child items for repositories as individual _Multibranch Pipelines_.
-
-Folder computation may happen automatically via webhook callbacks, as branches
-and repositories are created or removed.  Computation may also be triggered by
-the _Build Trigger_ defined in the configuration, which will automatically run a
-computation task after a period of inactivity (this defaults to run after one
-day).
-
+_Multibranch Pipeline_ projects and _GitHub Organization Folders_ extend Jenkins' folder functionality by introducing computed folders, which automatically manage their contents by dynamically creating and updating child items.
+ 
+* **Multibranch Pipeline Projects:** Previously, the computed folder functionality was located under the **Branch Indexing** tab, which included **Run Now** and **Log** options. Jenkins would generate child items for each eligible branch within a repository and automatically update the pipeline list as branches changed. In the updated version, these options are no longer under a specific tab and have been renamed to **Scan Multibranch Pipeline Now** and **Scan Multibranch Pipeline Log**.
+
+* **GitHub Organization Projects:** Originally, the computed folder functionality appeared under the **Folder Computation** tab, with **Run Now** and **Log** options. This functionality populated repositories as individual Multibranch Pipelines and provided insights into branch and repository scans. In the updated version, these options are no longer under a tab and have been renamed to **Scan Organization Now** and **Scan Organization Log**. 
+ 
+These updates followed the deprecation of the plugin:github-organization-folder[*GitHub Organization Folder*] plugin and its integration into the plugin:github-branch-source[*GitHub Branch Source*] plugin.
```

Comment:
delete

Reply:
Hi @timja,
Do you prefer the content to exclude information about the previous UI and the plugin deprecation, and instead focus solely on describing the current UI? Let me know your thoughts so I can adjust the content accordingly.

Reply:
Yes please current UI only, unless in a transition period not 8 years ago